play/main.ts

```diff
@@ -1,8 +1,12 @@
 import { createApp } from 'vue'
+import { ElSelect, ElOption } from '@element-plus/components'
 import App from './src/App.vue'
 
 import '@element-plus/theme-chalk/src/index.scss'
 
 const app = createApp(App)
 
+app.component(ElSelect.name, ElSelect)
+app.component(ElOption.name, ElOption)
+
```

Comment:
Remove this please. Actually we don't need to import and register components. `unplugin-vue-components` will import it automatically.